docs/modules/ROOT/pages/redis/reference.adoc

```diff
@@ -0,0 +1,24 @@
+= Redis reference
```

Comment:
I'd link to https://products.docs.vshn.ch/products/appcat/exoscale_dbaas.html#_redis to not duplicate data.

Reply:
I got rid of the reference doc and just linked to it from create. Does that make sense or do we still want a reference page?